test/fixtures/slack_identifiers.yml

```diff
@@ -0,0 +1,11 @@
+deployer:
+  identifier: 'U1234567890'
```

Comment:
These are referenced like `slack_identifiers(:deployer)`, which seems pretty clear to me. Is there another form that I haven't seen?